<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatwindowClass</class>
 <widget class="QWidget" name="chatwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatwindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="bgchatwindowframe">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QScrollArea" name="chatwindowbuttonscrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>43</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>43</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>336</width>
               <height>41</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>41</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="buttonlayout">
                <property name="sizeConstraint">
                 <enum>QLayout::SetMinimumSize</enum>
                </property>
                <item>
                 <widget class="QPushButton" name="pbmute">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Use this button to mute or unmute this user.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbbuddy">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Use this button to add this user to buddies or normals.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pblog">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Use this button to extract a logfile of the chat with htis user.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbidle">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Use this button to see the idle time of this user.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="lineWidth">
                   <number>2</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbctcp">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Use this button to handle your ctcp commands.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbstatus">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Use this button to see the status of this user.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbfilter">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Use this button to filter join,part,quit information.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbsmileys">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="chat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="send">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
